### PR TITLE
Added git lesson first draft

### PR DESCRIPTION
**Pull request by nmadd**
Still need to fill out a couple sections including the second exercise

### CODE REVIEW

File: unit1-intro-to-programming-in-javascript/git.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+* 11:30 - 1:00 Exercise 1
+* 1:00 - 2:00 Lunch
+* 2:00 - 2:55 Lecture part 2
+* 2:55 - 3:00 Break
+* 3:00 - 4:55 Exercise 2
+* 4:55 - 5:00  Break
+* 5:00 - 6:00 - Review 
+
+## Resources
+TODO: Add links for slides, live coding examples, helpful resources, etc.
```
Comment (bensussman) [inline unit1-intro-to-programming-in-javascript/git.md:22]: - https://www.sbf5.com/~cduan/technical/git/
- http://rypress.com/tutorials/git/index

**Comment by bensussman** [comment]
👍  I like it!

My advice is:
- Lean on other resources for content that explains the material instead of writing it all yourself
- Spend more time on the exercises and less on the lecture (in the content-creation-phase)
- Keep being awesome!

My concern right now is just the velocity that we are writing this content. I think we might want to lower the quality bar a bit (or steal more) so we can ship more content faster!! We'll both be very busy next week, and after that it's 3 short weeks to launch!!

**Comment by nmadd** [comment]
Yeah definitely agree. I'd love to be able to have this level of detail but it just takes too long. Rina had a good suggestion to have the students research and define a lot of the basic syntax and important terms on their own, instead of us writing it out and handing it to them. Would have the added benefit of helping teach them how to effectively learn on their own, plus we can have them submit it in a Google Form or something similar so we can track data. That combined with providing pre-reading/helpful resources should be pretty effective and help us crank out these lesson plans a lot faster.

Want to go ahead and merge this? I'm going to move on to other lessons for now but will come back and readjust this lesson once I have a better feel for the right length and structure

**Comment by bensussman** [comment]
Sure I can merge.

It looks like there are still two major TODOs though. Should you just delete those sections? Or come back later to with interesting exercises? I would love to have a merge conflict exercise as partners where the partners make conflicting changes, but then together they have to resolve the conflict (we can spoon feed them the conflict, but let them figure out how to resolve)

My concern with letting them "learn it on their own" is that we are overestimating their self-actualization skills and internet research skills, especially at this early stage of the program.

**Pull request merged**